tests/micro/zephyr/README.md

```diff
@@ -40,3 +40,9 @@ To see the list of supported values for `--board`, run:
 ```
 $ pytest test_zephyr.py --help
 ```
+
+If you like to test with a real hardware, you have the option to pass the serial number
```

Comment:
nit:

```suggestion
If you like to test with real hardware, you have the option to pass the serial number
```

Reply:
I'll apply it to the next PR if the CI passed.